doc/api/net.md

```diff
@@ -82,13 +107,19 @@ This becomes `null` when sending a socket to a child with
 connections use asynchronous `server.getConnections` instead.
 
 ### server.getConnections(callback)
+<!-- YAML
+added: v0.9.7
+-->
 
 Asynchronously get the number of concurrent connections on the server. Works
 when sockets were sent to forks.
 
 Callback should take two arguments `err` and `count`.
 
 ### server.listen(handle[, backlog][, callback])
+<!-- YAML
+added: v0.3.2
+-->
```

Comment:
I *think* v0.5.10 might be more appropriate here, when `net_uv.js` and `net_legacy.js`… um, converged, or something?